.sync.yml

```diff
@@ -1,6 +1,8 @@
 ---
 .travis.yml:
   secure: "RrAmWtM6eTjZZzD954AIgR179Pqp14lzHhd/C9cpKbTPynLncuim08CEvjmq+7pgAy9XDg1d02x3udfZt4btR1sBdyNRpNN3yUhWptmGU61HRJdiZq+nSLQkIYsqXanhk+O3NndFojO58WRD01dkWEc6HRHjlivuYNxDXmMkg9k="
+  docker_sets:
+  - set: docker/ubuntu-16.04
```

Comment:
Did you test if our centos/ubuntu14.04/debian8/debian9 (voxpupuli/modulesync_config#388) work as well?

Reply:
Only enabled this one because of testing

Reply:
Adding more of the docker sets now since i solved the issue with splunkserver running inside a docker container.